### ISSUE TITLE
Allow secrets to be read from files

### ISSUE THREAD

**Issue by ejj** [kelda/kelda#1376]
Philip Weaver on slack noted that his private tls keys exist in a file on disk.  Passing those in to `kelda secrets` is a pain, requiring some bash wizardry to cat the file and pipe it into the utility.  As this will be a common problem, making it possible to read from a file will be useful.

**Comment by ejj** [comment]
@kklin Do you agree with this issue?

**Comment by kklin** [comment]
Sounds fine. I want to be wary of adding flags but this seems appropriate/common

**Comment by kklin** [comment]
For the record, the current fix is `kelda secret secretName "$(cat fileOnDisk.txt)"`

**Comment by ejj** [comment]
One thing I've seen before from some tools, is to always read it from a file, and if `-` is passed in have that signal that it should be read from stdin.

**Comment by kklin** [comment]
Do those tools support setting the values directly too? (`kelda secret secretName secretValue`)

It sounds confusing to me. It would make sense if the only two inputs were from disk or stdin, but I think it's too much if the inputs are disk, stdin, or argument.

**Comment by ejj** [comment]
Yeah I can see that supporting all three is confusing.  Of the three, I'm not entirely convinced that passing it in on the command line is the one we should preserve.  There's some awkward properties of that ... notably it gets saved in the bash history (which isn't ideal for secrets).

Anyways, whoever fixes this issue can do some research into the use cases and figure out what's best.

**Comment by kklin** [comment]
That sounds reasonable. Don't need to figure it out now
